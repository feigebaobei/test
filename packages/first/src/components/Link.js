// function
import React, {
    // useState,
    useEffect,
    Fragment,
    useReducer
} from "react";
// import First from "./components/First";

export default function ReactComp () {
    let [state, dispatch] = useReducer((state, action) => {
        let res = state
        switch (action.type) {
            case '':
                break
            default:
                break
        }
        return res
    }, {})
    useEffect(() => {}, [])
    return (<div>
            from ReactComp
        </div>)
}