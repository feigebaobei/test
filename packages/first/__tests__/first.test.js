import React from 'react';
import renderer from 'react-test-renderer';
import Link from '../src/components/Link.js';

it('renders correctly', () => {
  const tree = renderer
    // .create(<Link page="http://www.facebook.com">Facebook</Link>)
    .create(<Link/>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});