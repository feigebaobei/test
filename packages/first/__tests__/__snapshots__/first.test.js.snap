// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  from ReactComp
</div>
`;
